### DISCUSSION TITLE
can i use endDayOfWeek?

### DISCUSSION THREAD

**Discussion by chlgbdud** [nhn/tui.calendar#1251] [Q&A] [answered]
document isn't have this option,

but i need to show only mon, tue, wed, thur data. how can i show these days?

**Comment by adhrinae** [comment, marked as answer]
No, you can't.
Perhaps you might need to use a different library like [fullcalendar](https://fullcalendar.io/).

It provides the `hiddenDays` option.

https://fullcalendar.io/docs/hiddenDays

**Reply by chlgbdud** [reply]
ty
